### ISSUE TITLE
Freature request: Watch expressions

### ISSUE THREAD

**Issue by netbrain** [Kindari/SublimeXdebug#71]
Add a way to set watch expressions. I work quite a bit with legacy code, and the context could hold several hundred variables. Watch expressions would be a welcomed feature to limit the variable list.

**Comment by martomo** [comment]
Unfortunately the 'watch' breakpoint types are not supported by DBGP protocol. ([source](http://xdebug.org/docs-dbgp.php#error-codes))

> 201 - breakpoint type not supported (for example I don't support
> 'watch' yet and thus return this error)

**Comment by cstuder** [comment]
Hmm, then how does Eclipse do it (Using xdebug with the PDT package)?

**Comment by martomo** [comment]
@cstuder What you are talking about is a [conditional breakpoint](http://xdebug.org/docs-dbgp.php#id1), you are able to define an expression for a breakpoint when to stop.

For example you set a breakpoint in **C:\wamp\www\wordpress\wp-settings.php** on line **72** and you only want to stop the script when the expression `$wp_version=='3.5.2'` is true.

```
breakpoint_set -i 7 -n 72 -t line -f file%3A//C%3A%5Cwamp%5Cwww%5Cwordpress%5Cwp-settings.php -- JHdwX3ZlcnNpb249PSczLjUuMic=
```

**Comment by cstuder** [comment]
@martomo: No, we're talking about manually entered watch expressions. These expressions are evaluated every time we stop at a breakpoint:

![watchexpressions](https://f.cloud.github.com/assets/288493/876959/c366d8dc-f8e3-11e2-899d-a33882809043.png)

They can be very, very useful, especially when the local scope is getting out of hand.

**Comment by martomo** [comment]
@cstuder I have added an initial commit martomo/SublimeTextXdebug@9484168838051f29123b8b6c24eb4b9911a3e776 for this feature to [SublimeTextXdebug](https://github.com/martomo/SublimeTextXdebug), please let me know what you think.
I still have to fine tune it some more, enable/disable watch expressions, remove single watch expression instead of having to clear/delete them all.
